app/components/graph-editor/graph-editor.component.ts

```diff
@@ -334,6 +334,18 @@ export class GraphEditorComponent implements AfterViewInit {
         this.redrawDelegate();
     }
 
+    copy(): void {
+        console.log("Editor copy");
+    }
+
+    cut(): void {
+        console.log("Editor cut");
+    }
+
+    paste(): void {
+        console.log("Editor paste");
+    }
```

Comment:
These might not be necessary in the editor once the new implementation is rolled out. You have access to the collection of selected entities from the `DrawableGraph` object now as well as its validator, create, and delete methods.